lib/nerves_motd.ex

```diff
@@ -93,8 +95,19 @@ defmodule NervesMOTD do
       [{"Hostname", hostname()}, {"Load average", load_average()}],
       []
     ] ++
-      ip_address_rows() ++
-      Keyword.get(opts, :extra_rows, [])
+      ip_address_rows() ++ extra_rows(opts)
+  end
+
+  @spec extra_rows([option()]) :: IO.ANSI.ansidata()
+  defp extra_rows(opts) do
+    case opts[:extra_rows] do
+      fun when is_function(fun, 0) -> fun.()
```

Comment:
```suggestion
      fun when is_function(fun, 0) -> fun.()
      {m, f, args} -> apply(m, f, args)
```
I was thinking that if you're adding functions, might as well add MFargs since those are sometimes nicer for the `config.exs`.

Reply:
I can do that. I couldn't think of a reason to need args since what is usually available on config is available at runtime. And with 0 arity you can make your module anonymous with `&SomeModule.rows/0`

Reply:
I agree that the function ref way doesn't need arguments.